src/tests/system/tests/test_cli.py

```diff
@@ -0,0 +1,41 @@
+"""
+Authselect Command Line Interface Tests
+
+:requirement: Authselect replaced authconfig
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.Client)
+def test_cli__is_feature_enabled(client: Client):
+    """
+    :title: Authselect local profile is functionally tested
+    :setup:
+        1. Create local user "user-1"
+        2. Select local profile
+        3. Configure SSSD for local proxy provider
+        4. Start SSSD
+    :steps:
+        1. Authenticate as "user-1"
+    :expectedresults:
+        1. Authentication is successful for "user-1"
+    :customerscenario: False
```

Comment:
This looks like copy/paste error. The test looks good but the description is for something else.